### ISSUE TITLE
Regularize POS tag for Christos

### ISSUE THREAD

**Issue by amir-zeldes** [CopticScriptorium/tagger-part-of-speech#6]
Should probably be N since it appears with an article ('the messiah') - currently most corpora have N more than NPROP

**Comment by ctschroeder** [comment]
Ok. I will add issues to corpora where it's NPROP.  Do we need to change 
anything else? Thank you.

**Comment by amir-zeldes** [comment]
No, it's just an individual thing I noticed, but I am willing to bet there are plenty of other little inconsistencies... We could try to automatically find them in the future, but there are too many other open threads at the moment!

**Comment by ctschroeder** [comment]
- [x] Besa (food, aphthonia)

- [x] 1cor (15, 16)

- [x] abraham

- [x] eagerness

**Comment by ctschroeder** [comment]
an updated search list https://annis.copticscriptorium.org/annis/?id=b87f3bc4-c569-43e2-b1c4-c8b66d72b68d

@amir-zeldes one of the works affected is john.constantinople (penitence.01). When I went in to edit, I noticed the spreadsheets for both penitence.01 & 02 were empty. Before I import the sgml from the last commit to github, I wanted to check with you to be sure there isn't some reason I shouldn't.

**Comment by ctschroeder** [comment]
Oh dear proclus.23 in proclus.homilies is empty. creating new issue

**Comment by ctschroeder** [comment]
[empty spreadsheet issue](https://github.com/CopticScriptorium/corpora/issues/150)
